README.md

```diff
@@ -1,72 +1,77 @@
-# Author - Varun Sree Bholalayam (00807289)
+# Deep learning vs traditional ML algorithms for AMR detection
+
+Master Thesis Proposal
+
+Author - Varun Sree Bholalayam (00807289)
 
 
 ## Problem Introduction
-***********************
-Antimicrobial Resistance (AMR) occurs when bacteria, viruses, fungi and parasites evolve over time and no longer respond to medicines making infections harder to treat and increasing the risk of disease spread, severe illness and death. AMR is a much overlooked aspect in the field of public health since the predicted mortality rate due to AMR by the year 2050 is more concerning than cancer<sup>1</sup>.
 
-Some examples of so called 'superbugs' are,
+Antimicrobial Resistance (AMR) occurs when bacteria, viruses, fungi and parasites evolve over time and no longer respond to medicines making infections harder to treat and increasing the risk of disease spread, severe illness and death. AMR is a much overlooked aspect in the field of public health since the predicted mortality rate due to AMR by the year 2050 is more concerning than cancer<sup>1</sup>.
 
-1.Drug resistant tuberculosis. 
+Some examples of so called "superbugs" are:
 
-2.Methicillin-resistant Staphylococcus aureus.  
+1. Drug resistant tuberculosis
+2. Methicillin-resistant Staphylococcus aureus
+3. Multidrug-resistant Pseudomonas aeruginosa.
 
-3.Multidrug-resistant Pseudomonas aeruginosa.  
+Both genotypic and phenotypic drug susceptibility tests (DST) are used clinically. Phenotypic DST have limitations including extended turnaround time for slow-growing bacteria such as Mycobacterium tuberculosis (MTB) and bias due to potential contamination<sup>2</sup>.
 
-Both genotypic and phenotypic drug susceptibility tests are used clinically. Phenotypic DST have limitations including extended turnaround time for slow-growing bacteria such as Mycobacterium tuberculosis (MTB) and bias due to potential contamination<sup>2</sup>.
+<!-- I don't know how to connect the previous paragraph -->
 
 Currently tools such as ARIBA, Resfinder and databases such as CARD, ARDB are used to analyse whether a known AMR gene is present in the sequenced data of a bacteria using various assembly and mapping algorithms. Although these tools are efficient in finding the presence of known AMR genes, the need of the hour demands tools that can predict whether a known first line drug is effective in treatment or not. Rule-based predictors exist such as Mykrobe predictor which uses De Bruijn graph representation of bacterial diversity to identify species and resistance profiles of clinical isolates<sup>3</sup>. Since genetic mechanisms that result in evolution of AMR genes are rapid, the competence of deterministic AI models should be scrutinized. A study published in 2022 Feb 14 is brought to the spotlight here<sup>2</sup>.
 
+<!-- AI models should be scrutinizied => isn't that a proposal? (compared to the problem) -->
```

Comment:
AI model mentioned here  is not machine learning model. The tool called Mykrobe uses rule based prediction. Study says that machine learning model has a slight upper-hand over this tool.  What I meant was, to scrutinise this  deterministic AI approach used by Mykrobe.